docs/model_pipeline.rst

```diff
@@ -93,9 +105,35 @@ Time series
    :file: ../oemof_b3/schema/timeseries.csv
 
 
-A few more conventions are important to know. Missing data is left empty. If a value applies to all
-regions, this is indicated by :attr:`ALL`. If it applies to the sum of regions, by :attr:`TOTAL`.
-There is no unit transformation within the model, i.e. the user needs to ensure the consistency of units.
+.. _conventions_resources_label_:
+
+Conventions
+^^^^^^^^^^^
+
+A few more conventions are important to know:
+
+* Missing data is left empty.
+* There is no unit transformation within the model, i.e. the user needs to ensure the consistency of units.
+
+* The parameters :attr:`id_scal` and :attr:`id_ts` are optional and will be added automatically if you do not specify them.
+
+* The parameter :attr:`scenario_key` allows you to pass values that are associated with a specific scenario.
+
+* The parameter :attr:`name` must be specified in a certain fixed concatenation of parameters: :attr:`region`-:attr:`carrier`-:attr:`tech` (example: :attr:`B-biomass-g`).
+
+* If a value applies to all regions, :attr:`region` corresponds to :attr:`ALL` for this value.
+* If :attr:`region` is set to :attr:`ALL` in the model-own assumptions, :attr:`name` is to be left blank. The name will be automatically added per region modelling the energy system.
+* If a value applies to the sum of regions, :attr:`region` corresponds to :attr:`TOTAL` for this value.
+
+* Different attributes can be set for :attr:`var_name`. Which component is assigned to which attributes can be found in chapter `Overview <https://oemoflex.readthedocs.io/en/latest/overview.html>`_ of the :attr:`oemoflex` documentation.
```

Comment:
There is a connection between the var_names and the attributes of the components, but not all var_names are attributes of components. I would drop the sentence here.

Reply:
> There is a connection between the var_names and the attributes of the components, but not all var_names are attributes of components. I would drop the sentence here.

You mean `capacity_cost_overnight`, `lifetime`, `fixom_cost`, `storage_capacity_cost_overnight`, `storage_fixom_cost` and `wacc`?
But surely it needs some kind of guidance so that `var_name` is assigned correctly in the input data? Would it be a solution to add this to the description above?

"...the :attr:`oemoflex` documentation. In addition, oemof-B3 can calculate annuized costs. For this, additional attributes must be passed with :attr:`var_name`: attr:`capacity_cost_overnight`, :attr:`storage_capacity_cost_overnight`, attr:`wacc`, attr:`lifetime` and attr:`fixom_cost."

Reply:
Clarified with commit https://github.com/rl-institut/oemof-B3/pull/280/commits/153339fb87c76fc68bea301023529bf46b3e418b.